drivers/supplemental_enrollment_data/config/routes.rb

```diff
@@ -0,0 +1,6 @@
+BostonHmis::Application.routes.draw do
+  scope :supplemental_enrollment_datum do
+    # TODO
+    # get '/my_path', to: 'supplemental_enrollment_data/my_controller'
+  end
```

Comment:
Do empty route scopes affect the app at all?

Reply:
they shouldn't, but I'll pull that.